src/goose/brokers/redis/commands.clj

```diff
@@ -155,6 +155,28 @@
 (defn del-from-list [conn list element]
   (wcar* conn (car/lrem list 1 element)))
 
+(defn list-position-multiple [conn list elements]
+  (let [response (wcar* conn (doseq [e elements]
+                               (car/lpos list e)))]
+    (if (= (count elements) 1)
+      (vector response)
+      response)))
```

Comment:
```suggestion
    (if (= (count elements) 1)
      {elements response}
      (zipmap elements response))))
```

Reply:
This doesn't make sense since job/position terminology doesn't make sense inside redis commands.